packages/react-data-grid/src/ReactDataGrid.js

```diff
@@ -558,10 +558,9 @@ class ReactDataGrid extends React.Component {
   };
 
   getHeaderRows = () => {
-    let rows = [{ ref: (node) => this.row = node, height: this.props.headerRowHeight || this.props.rowHeight, rowType: HeaderRowType.HEADER }];
+    const rows = [{ height: this.props.headerRowHeight || this.props.rowHeight, rowType: HeaderRowType.HEADER }];
```

Comment:
`ref: this.row` is not used anywhere. Header row defines its own [ref](https://github.com/adazzle/react-data-grid/blob/master/packages/react-data-grid/src/Header.js#L107).